# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apache/hotfix.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -p -up httpd-2.2.17/server/core_filters.c.orig httpd-2.2.17/server/core_filters.c
--- httpd-2.2.17/server/core_filters.c.orig	2010-11-27 01:33:03.000000000 -0200
+++ httpd-2.2.17/server/core_filters.c	2010-11-27 01:34:11.000000000 -0200
@@ -83,6 +83,7 @@ do { \
  *
  * XXXX: Should this function be added to APR-Util?
  */
+__attribute__((__noinline__))
 static void brigade_move(apr_bucket_brigade *b, apr_bucket_brigade *a,
                          apr_bucket *e)
 {
